app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -64,7 +64,7 @@ data class HikeCardStyleProperties(
 fun HikeCard(
     title: String,
     modifier: Modifier = Modifier,
-    elevationData: List<Double> = emptyList(),
+    elevationData: List<Double>?,
```

Comment:
Discussion: I was thinking about whether it makes more sense to pass the data directly here or to pass the viewmodel and have the composable fetch the data from the viewmodel, but I think the way you solved it works well, since this version is simpler and more easy to test. Have you thought about this?